portlets/src/main/webapp/vue-app/realizations/components/Realizations.vue

```diff
@@ -283,6 +283,9 @@ export default {
         this.getRealizations(false);
       } );
     },
+    getReport() {
+      return this.$realizationsServices.getReport(this.fromDate, this.toDate,0, 100);
```

Comment:
Do we need this limit ? Why is it set to 100 ?

Reply:
done !